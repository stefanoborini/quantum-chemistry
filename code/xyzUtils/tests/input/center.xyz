2
hello
C 0.0 0.0 1.0
H 0.0 0.0 0.5
2
hello
C 0.0 0.0 1.0
C 0.0 0.0 0.5
