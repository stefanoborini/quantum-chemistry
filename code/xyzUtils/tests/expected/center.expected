 2
hello
C     0.00000    0.00000    0.03874
H     0.00000    0.00000   -0.46126
 2
hello
C     0.00000    0.00000    0.25000
C     0.00000    0.00000   -0.25000
